app/models/polling/scheduler.rb

```diff
@@ -19,8 +19,10 @@ def mode(mode, time = Rails.configuration.scheduler_polling_interval)
     end
 
     def perform(action = -> {})
-      @job = @rufus_scheduler.method("schedule_#{@mode}")
-                            .call(time, **opts, &action)
+      #ActiveRecord::Base.connection_pool.with_connection do
```

Comment:
Is this commented-out code intentional? Is the pool used by default anyway? Either way, some change required here.

Reply:
Thanks, I didn't intend to remove this, it is there to ensure that any connection used from the pool is explicitly released when it's done.